libs/blockscout-db/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "blockscout-db"
-version = "4.1.8"
+version = "5.0.0"
```

Comment:
version of crate should be the same as version of blockscout (6.8.0)

Reply:
hmmm... what if a breaking change is introduced but blockscout's major version is not changed? i.e. github.com/blockscout/blockscout-rs/pull/1051

Reply:
how about using `5.0.0+6.8.0` as the version?

Reply:
makes sense, @sevenzing do you agree?

Reply:
we could even do smth like "5.0.0+blockscout-6.8.0" or "5.0.0+blockscout.6.8.0" or "5.0.0+blockscout6.8.0"